### ISSUE TITLE
“Introduction to automated testing” is hard to follow; needs clear instructions that assume the reader is carrying out this task for the very first time

### ISSUE THREAD

**Issue by grace-anderson** [mdn/content#15272]
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Automated_testing

### What specific section or headline is this issue about?

starting at html-tidy heading the instructions start to lose clarity

### What information was incorrect, unhelpful, or incomplete?

Not clear which file you are editing, and where you are adding or updating within the file

Export the html task using:
exports.html = html;

Change the default export to:
exports.default = html;

### What did you expect to see?

clear instructions that assume the reader is carrying out this task for the very first time. 

Make sure such instructions are ELI5, and keep setting the reader into the context - for e.g. what will the gulpfile.js look like before this step then after this step (or whatever file you're referring to here because I got completely lost) 

Example of other ambigious statements that would lose the reader:
"You can try your own test content if you like, but bear in mind that such tools won't work on internal JS/CSS — you need external file" What does this mean - what is the difference between an internal / external file. As a new programmer, how can I identify the difference

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/tools_and_testing/cross_browser_testing/automated_testing`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Automated_testing
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/tools_and_testing/cross_browser_testing/automated_testing/index.md
* Last commit: https://github.com/mdn/content/commit/d5d9a70d1f8bc041c4ff226c3ff7e02382c5efef
* Document last modified: 2022-04-14T03:51:37.000Z

</details>

**Comment by hamishwillee** [comment]
Thanks @grace-anderson . Yes. Would you like to take a shot an update?
